drivers/nrf9160_gps/nrf9160_gps.c

```diff
@@ -179,6 +221,10 @@ static void gps_thread(int dev_ptr)
 			break;
 
 		case NRF_GNSS_NMEA_DATA_ID:
+			if (operation_blocked) {
```

Comment:
I am checking with the GPS provider how PVT and NMEA notifications are tied together. The assumption here is that the previous PVT notification dictate the validity of the next NMEA notifications.

Reply:
I am not sure if we will have this information in resonable time for merging this PR. So I think this will be something we can update later.

Reply:
Alright, I'll leave it as is, then.